src/components/lobby/CreateRoomForm.js

```diff
@@ -54,7 +43,7 @@ const CreateRoomForm = ({ games }) => {
         value={players}
         onChange={(_, { value }) => setPlayers(value)}
       />
-      <Button fluid color="green" type="submit">
+      <Button fluid color="green" type="submit" disabled={userInGame || loading}>
```

Comment:
Why do we even let people see the lobby while in game? Maybe just force redirect them into room until they leave?

Reply:
ok, we can do that.
I just thought that sometimes users may want to see the Lobby while they are playing